contracts/root/RootChainManager/RootChainManager.sol

```diff
@@ -75,6 +75,7 @@ contract RootChainManager is IRootChainManager, Initializable, AccessControl {
         external
         only(DEFAULT_ADMIN_ROLE)
     {
+        require(newChildChainManager != address(0x0), "New child manager should not be zero");
```

Comment:
Revert message can be changed to match the format of other ones in this contract.

Reply:
Fixed.